wolfcrypt/src/asn.c

```diff
@@ -38280,6 +38280,41 @@ static int ParseCRL_AuthKeyIdExt(const byte* input, int sz, DecodedCRL* dcrl)
 }
 #endif
 
+#ifdef WOLFSSL_ASN_TEMPLATE
+
+static const ASNItem crlNumASN[] = {
+/* CRL NUM */          {0, ASN_INTEGER, 0, 0, 0 },
+};
+enum {
+    CRLNUMASN_IDX_CRL_NUM = 0
+};
+
+/* Number of items in ASN.1 template for CrlNumber. */
+#define crlNumASN_Length (sizeof(crlNumASN) / sizeof(ASNItem))
+
+static int ParseCRL_CrlNumExt(const byte* input, int sz, DecodedCRL* dcrl)
```

Comment:
Use GetShortInt() instead?

Reply:
Yes that would be cleaner. I did not realize GetShortInt internals had a leg for WOLFSSL_ASN_TEMPLATE. I will rewrite to use GetShortInt instead.